Source/WebCore/platform/graphics/GraphicsLayer.cpp

```diff
@@ -903,144 +903,144 @@ void GraphicsLayer::dumpProperties(TextStream& ts, OptionSet<LayerTreeAsTextOpti
-        ts << indent << "(acceleratesDrawing " << m_acceleratesDrawing << ")\n";
+        ts << indent << "(acceleratesDrawing "_s << m_acceleratesDrawing << ")\n"_s;
 
     if (options & LayerTreeAsTextOptions::IncludeBackingStoreAttached)
-        ts << indent << "(backingStoreAttached " << backingStoreAttachedForTesting() << ")\n";
+        ts << indent << "(backingStoreAttached "_s << backingStoreAttachedForTesting() << ")\n"_s;
 
     if (m_transform && !m_transform->isIdentity()) {
-        ts << indent << "(transform ";
-        ts << "[" << m_transform->m11() << " " << m_transform->m12() << " " << m_transform->m13() << " " << m_transform->m14() << "] ";
-        ts << "[" << m_transform->m21() << " " << m_transform->m22() << " " << m_transform->m23() << " " << m_transform->m24() << "] ";
-        ts << "[" << m_transform->m31() << " " << m_transform->m32() << " " << m_transform->m33() << " " << m_transform->m34() << "] ";
-        ts << "[" << m_transform->m41() << " " << m_transform->m42() << " " << m_transform->m43() << " " << m_transform->m44() << "])\n";
+        ts << indent << "(transform "_s;
+        ts << "["_s << m_transform->m11() << " "_s << m_transform->m12() << " "_s << m_transform->m13() << " "_s << m_transform->m14() << "] "_s;
+        ts << "["_s << m_transform->m21() << " "_s << m_transform->m22() << " "_s << m_transform->m23() << " "_s << m_transform->m24() << "] "_s;
+        ts << "["_s << m_transform->m31() << " "_s << m_transform->m32() << " "_s << m_transform->m33() << " "_s << m_transform->m34() << "] "_s;
+        ts << "["_s << m_transform->m41() << " "_s << m_transform->m42() << " "_s << m_transform->m43() << " "_s << m_transform->m44() << "])\n"_s;
     }
 
     // Avoid dumping the sublayer transform on the root layer, because it's used for geometry flipping, whose behavior
     // differs between platforms.
     if (parent() && m_childrenTransform && !m_childrenTransform->isIdentity()) {
-        ts << indent << "(childrenTransform ";
-        ts << "[" << m_childrenTransform->m11() << " " << m_childrenTransform->m12() << " " << m_childrenTransform->m13() << " " << m_childrenTransform->m14() << "] ";
-        ts << "[" << m_childrenTransform->m21() << " " << m_childrenTransform->m22() << " " << m_childrenTransform->m23() << " " << m_childrenTransform->m24() << "] ";
-        ts << "[" << m_childrenTransform->m31() << " " << m_childrenTransform->m32() << " " << m_childrenTransform->m33() << " " << m_childrenTransform->m34() << "] ";
-        ts << "[" << m_childrenTransform->m41() << " " << m_childrenTransform->m42() << " " << m_childrenTransform->m43() << " " << m_childrenTransform->m44() << "])\n";
+        ts << indent << "(childrenTransform "_s;
+        ts << "["_s << m_childrenTransform->m11() << " "_s << m_childrenTransform->m12() << " "_s << m_childrenTransform->m13() << " "_s << m_childrenTransform->m14() << "] "_s;
```

Comment:
Can we use `‘[‘` and the like for the single character ones? Maybe a global regular expression search and replace for `<< "(?)"_s` to `<< ‘\1’`. Unless TextStream doesn’t do `char`.

Reply:
Looks like it works. I'll fix.